docs/sdk/android/introduction.md

```diff
@@ -54,6 +54,18 @@ compileOptions {
 }
 ```
 
+Otherwise, for the file attachment to work properly, you must add a file named `file_paths.xml` in the `res/xml` directory of your app module.
```

Comment:
Quizás haría una sección para esto, tipo:
```
**Include file_paths.xml file for attachments:**
Since Android 11 you must add a file named `file_paths.xml` in the `res/xml` directory of your app module for the file attachment to work properly.
```